rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/model/RascalADTs.java

```diff
@@ -33,18 +33,18 @@ public class RascalADTs {
     private RascalADTs() {}
     public static class LanguageContributions {
         private LanguageContributions () {}
-        public static final String PARSER = "parser";
-        public static final String ANALYZER = "analyzer";
-        public static final String BUILDER = "builder";
-        public static final String OUTLINER = "outliner";
+        public static final String PARSER = "parsing";
+        public static final String ANALYZER = "analysis";
+        public static final String BUILDER = "build";
+        public static final String OUTLINER = "documentSymbol";
         public static final String LENS_DETECTOR = "lenses";
-        public static final String INLAY_HINTER = "inlayHinter";
-        public static final String COMMAND_EXECUTOR = "executor";
-        public static final String DOCUMENTER = "documenter";
-        public static final String DEFINER = "definer";
-        public static final String REFERRER = "referrer";
-        public static final String IMPLEMENTER = "implementer";
-        public static final String CODE_ACTION_CONTRIBUTOR = "actions";
+        public static final String INLAY_HINTER = "inlayHint";
+        public static final String COMMAND_EXECUTOR = "execution";
+        public static final String DOCUMENTER = "hover";
+        public static final String DEFINER = "definition";
+        public static final String REFERRER = "reference";
+        public static final String IMPLEMENTER = "implementation";
+        public static final String CODE_ACTION_CONTRIBUTOR = "codeAction";
```

Comment:
Should we rename the constants, too? I realize it's painful and invasive (iirc, there are also various methods that have these names), and for the Rascal *user* it doesn't matter. But for the Rascal *contributor* (especially new ones), it's nicer to work with a single style of names. I worry this may keep haunting us (not terribly, but still) if we don't unify as best as we can now. (I'd be happy to more actively contribute here btw, if you need the help.)

Reply:
Using java rename this shouldn't be too hard. Let's do it.

Reply:
@sungshik I welcome your help here. I'll be fixing stuff **/LanguageServer.rsc mostly.

Reply:
pls pull though. there was some unfinished business in the constructor names